llms/oobabooga_llm.py

```diff
@@ -7,26 +7,26 @@
 config = load_config()
 
 PARAMS = {
-    'max_new_tokens': 100,
-    'do_sample': True,
-    'temperature': 0.2,
-    'top_p': 0.73,
-    'typical_p': 1,
-    'repetition_penalty': 1.1,
-    'encoder_repetition_penalty': 1.0,
-    'top_k': 0,
-    'min_length': 0,
-    'no_repeat_ngram_size': 0,
-    'num_beams': 1,
-    'penalty_alpha': 0,
-    'length_penalty': 1,
-    'early_stopping': False,
-    'seed': -1,
-    'add_bos_token': True,
-    'truncation_length': 2048,
-    'ban_eos_token': False,
-    'skip_special_tokens': True,
-    'stopping_strings': [],
+    "max_new_tokens": 100,
+    "do_sample": True,
+    "temperature": 0.5,
```

Comment:
I’m a bit lost at the parameters, why do we set a temperature here at 0.5 and in _call function it’s 0.2?

Reply:
i was playing with the temperature and did not set it back in the default. A default at 0.5 is fine anyway, and we'll want to expose a way to override the temperature when initializing `OobaboogaLLM()` instead.